gateway/node.go

```diff
@@ -214,6 +216,20 @@ func (gw *Node) checkTipsetKey(ctx context.Context, tsk types.TipSetKey) error {
 }
 
 func (gw *Node) checkTipset(ts *types.TipSet) error {
+	// if mir consensus, we check a height diff, not actual timestamps
+	if global.IsConsensusAlgorithm(global.MirConsensus) {
```

Comment:
Potentially stupid question but how/where are we setting this? The gateway can be pointed at any RPC and has its own config file/dir so it wouldn't know how eudico was launched, I think?